# Differet variants on chrM between mother and child

Based on the PE sequencing reads, call and annotate the variants on chrM, and compare them between mother and child. You can use one mother-child pair or more paris. Try to find some interesting variants or genes.

## Data and reference
* [ESHG 3: Variant Calling](https://test.galaxyproject.org/u/galaxyproject/h/eshg-3-variant-calling-1)
* [Index of /~anton/share/mt.data](http://www.bx.psu.edu/~anton/share/mt.data/)
* [Data Library “Mt Study 2014”](https://usegalaxy.org/library_common/browse_library?show_deleted=False&cntrller=library&use_panels=False&id=f11cf06cdcc13cfd)

---

# Find differential expression genes between adrenal and brain

Datasets are paired-end 50bp reads from adrenal and brain tissues. The sampled reads map mostly to a 500Kb region of chromosome 19, positions 3-3.5 million (chr19:3000000:3500000). Based on the sequencing reads, find and annotate/analyze differential expression genes between adrenal and brain. You can also try to find novel transcripts/isoforms.

## Data and reference
* [RNA-seq Analysis Exercise](https://usegalaxy.org/u/jeremy/p/galaxy-rna-seq-analysis-exercise)

---

# Find differential expression genes between MeOH and R3G

Datasets are paired-end reads for conditions MeOH and R3G. Each condition contains three replicates. The samples reads map mostly to chr12. Based on the sequencing reads, find and annotate/analyze differential expression genes between MeOH and R3G. You can also try to find novel transcripts/isoforms.

## Data and reference
* [Aligning PE RNA-Seq Reads to a Genome](http://bioinformatics.ucdavis.edu/docs/2014-december-workshop/Thursday_MB_Tophat_PE.html)
* [Cam RNA-Seq Day 1](https://test.galaxyproject.org/u/galaxyproject/h/cam-rna-seq-day-1)
* [Data Library “Training”](https://test.galaxyproject.org/library_common/browse_library?show_deleted=False&cntrller=library&use_panels=False&id=d1d508fb296552d5)

---

# Comparing the binding profiles of Nanog and Pou5f1 (Oct4) in h1-ESC cells

Datasets are reads from 32.8 Mb of chromosome 12 (chr12:1,000,000-33,800,000). Each condition contains two replicates. Based on the sequencing reads, compare the binding profiles of two transcription factors, Nanog and Pou5f1 (Oct4) in h1-ESC cells. Try to find the differential binding between two conditions and do the Enrichment analysis.

## Data and reference
* [ChIP-Seq Data Analysis using Galaxy](https://cgrlucb.wikispaces.com/ChIP-Seq+Data+Analysis+using+Galaxy)
* [Introduction to ChIP-Seq](http://bioinformatics.sph.harvard.edu/ngs-workshops/courses/introduction-to-chip-seq/)
* [Tanta Bic workshop Source history](https://usegalaxy.org/u/hanaa/h/source-history)
* [Data Library “Training”](https://test.galaxyproject.org/library_common/browse_library?show_deleted=False&cntrller=library&use_panels=False&id=d1d508fb296552d5)

---

# Comparing the binding sites of CTCF in G1E_ER4 and G1E cell lines

G1E is a model for erythropoiesis, the G1E line is a GATA1 null derived line which can be induced to differentiate by estradiol treatment (thus G1E-ER4). Datasets contain only reads aligning to chr19. Based on the sequencing reads, inspect differential CTCF binding sites between two differentiation time points. Try to annotate these binding sites and associated genes.

## Data and reference
* [ChIP-seq exercises](https://usegalaxy.org/u/james/p/exercise-chip-seq)

