# QC and preprocess
## Data with workflow
* [Manipulation of FASTQ data with Galaxy](http://hpc.ilri.cgiar.org/training/Glossina_annotation/Galaxy_Intro.html)



---



# Exome-Seq

## Data

* [Index of /~anton/share/mt.data](http://www.bx.psu.edu/~anton/share/mt.data/)


* [Data Library “Mt Study 2014”](https://usegalaxy.org/library_common/browse_library?show_deleted=False&cntrller=library&use_panels=False&id=f11cf06cdcc13cfd)

## Workflow
* [sahilseth/flowr](https://github.com/sahilseth/flowr/tree/master/inst/pipelines)
* [Galaxy Workflow 'Exome Basics Analysis'](https://usegalaxy.org/workflow/display_by_username_and_slug?username=jeremy&slug=exome-basics-analysis)
* [Galaxy Workflow 'Tumor Exome Analysis'](https://usegalaxy.org/workflow/display_by_username_and_slug?username=jeremy&slug=tumor-exome-analysis)
* [Galaxy Workflow 'Exome Analysis'](https://usegalaxy.org/u/jeremy/w/exome-analysis)

## Data with workflow
### nkatiyar
* [Galaxy_workshop](http://biocluster.ucr.edu/~nkatiyar/Galaxy_workshop/)
* [Galaxy_workshop/Snpseq](http://biocluster.ucr.edu/~nkatiyar/Galaxy_workshop/Snpseq/)
### NA12878

* [ssadedin/variant_calling_pipeline](https://github.com/ssadedin/variant_calling_pipeline)
### Mother vs. child
* [ESHG 3: Variant Calling](https://test.galaxyproject.org/u/galaxyproject/h/eshg-3-variant-calling-1)




---



# RNA-Seq
## Workflow
* [UAB Galaxy RNA Seq Step by Step Tutorial](https://docs.uabgrid.uab.edu/wiki/UAB_Galaxy_RNA_Seq_Step_by_Step_Tutorial)
* [Aligning SE RNA-Seq Reads to a Reference Genome - exercises](http://bioinformatics.ucdavis.edu/docs/2014-december-workshop/Thursday_MB_Tophat_Model_SE.html)
* [Aligning PE RNA-Seq Reads to a Genome - exercises](http://bioinformatics.ucdavis.edu/docs/2014-december-workshop/Thursday_MB_Tophat_PE.html)

## Data with workflow
### GM12878 vs h1-hESC
* [RNA-seq Analysis Exercise](http://hpc.ilri.cgiar.org/training/Glossina_annotation/RNA-Seq.html)
* [Galaxy Tutorial 2](https://usegalaxy.org/u/penelope22hart/h/tutorial-2)
* [Index of /training/Glossina_annotation/RNA-Seq_files](http://hpc.ilri.cgiar.org/training/Glossina_annotation/RNA-Seq_files/)
### nkatiyar
* [Galaxy_workshop](http://biocluster.ucr.edu/~nkatiyar/Galaxy_workshop/)
* [Galaxy_workshop/Rnaseq](http://biocluster.ucr.edu/~nkatiyar/Galaxy_workshop/Rnaseq/)
### adrenal vs. brain
* [RNA-seq Analysis Exercise](https://usegalaxy.org/u/jeremy/p/galaxy-rna-seq-analysis-exercise)
### MeOH
* [Cam RNA-Seq Day 1](https://test.galaxyproject.org/u/galaxyproject/h/cam-rna-seq-day-1)




---



# ChIP-Seq
## Workflow
* [ChIP-Seq Analysis - exercise](http://bioinformatics.ucdavis.edu/docs/2014-december-workshop/Wednesday_NJ_Chip_Seq_Exercise.html)
* [ChIP-Seq – Quality filtering, read mapping and Peak annotation](http://jvanheld.github.io/cisreg_course/chip-seq/practical/chip-seq.html)
* [Galaxy Workflow 'ChIP-seq analysis (create BAM, bigWig and PeakCalling)'](https://usegalaxy.org/u/chip-seq-helin-group/w/mmusculus-mm10-create-bam-bigwig-and-peakcalling-for-chip-seq)

## Data with workflow
### h1-ESC (Nanog vs. Pou5f1 vs. control)
* [ChIP-Seq Data Analysis using Galaxy](https://cgrlucb.wikispaces.com/ChIP-Seq+Data+Analysis+using+Galaxy)
* [Introduction to ChIP-Seq](http://bioinformatics.sph.harvard.edu/ngs-workshops/courses/introduction-to-chip-seq/)
* [Tanta Bic workshop Source history](https://usegalaxy.org/u/hanaa/h/source-history)
### G1E_ER4 (CTCF in G1E_ER4 vs. G1E)

* [ChIP-seq exercises](https://usegalaxy.org/u/james/p/exercise-chip-seq)




---



# Others
## Summer school
* [Analysis of Cancer Genomes (cruk-summer-school-2016)](https://bioinformatics-core-shared-training.github.io/cruk-summer-school-2016/)
* [Analysis of Cancer Genomes (bioinformatics-core-shared-training/cruk-summer-school-2016)](https://github.com/bioinformatics-core-shared-training/cruk-summer-school-2016)
* [Best Practices in the analysis of RNA-seq and ChIP-seq data (bioinformatics-core-shared-training/cruk-bioinf-sschool)](https://github.com/bioinformatics-core-shared-training/cruk-bioinf-sschool)
* [Welcome to the June 2015 Workshop - Using the Linux Command-Line for Analysis of High Throughput Sequence Data](http://bioinformatics.ucdavis.edu/docs/2015-june-workshop/index.html)
* [Welcome to the June 2016 Workshop - Using Galaxy for Analysis of RNA-Seq and ChIP-Seq Data](http://bioinformatics.ucdavis.edu/docs/2016-june-workshop/index.html)

## Data collections
* [Data Library “Training”](https://test.galaxyproject.org/library_common/browse_library?show_deleted=False&cntrller=library&use_panels=False&id=d1d508fb296552d5)




---



# Bad links
* [Data Analysis Example Data](http://www.omixon.com/data-analysis-example-data)
* [ChIP-Seq: quality filtering, read mapping and peak-calling](http://pedagogix-tagc.univ-mrs.fr/courses/ASG1/practicals/chip-seq/mapping_tutorial.html)


